ci/install-dependencies.sh

```diff
@@ -11,6 +11,16 @@ UBUNTU_COMMON_PKGS="make libssl-dev libcurl4-openssl-dev libexpat-dev
  tcl tk gettext zlib1g-dev perl-modules liberror-perl libauthen-sasl-perl
  libemail-valid-perl libio-socket-ssl-perl libnet-smtp-ssl-perl"
 
+install_sparse () {
+	local urlbase=https://dev.azure.com/git/git/_apis/build/builds
+	local id=$(curl "$urlbase?definitions=10&statusFilter=completed&resultFilter=succeeded&\$top=1" | jq '.value[].id')
```

Comment:
note for myself: Add `-S`

Reply:
@dscho I think the sparse from Azure is either too old or too new. It produce some false positive in system header. v0.6.1 doesn't have this problem.

I don't have any Ubuntu system to check the deb file.

Reply:
Isn't it built from the latest `master`? I'm on my phone for another 4h and have a hard time to check.

Reply:
I pulled 2 docker image for `ubuntu:xenial` and `ubuntu:bionic`,
it looks like latest sparse have false positive with `ubuntu:bionic`

I'll look into it further.

Reply:
I guess https://lore.kernel.org/git/20200522002502.9010-1-luc.vanoostenryck@gmail.com/t/#u addresses that?

Reply:
@dscho Could you please check `gcc` version on Azure pipeline?

From this conversation: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=505177#15

I *think*
Sparse need to be built with the same compiler version with the one it's used to check. Otherwise, `sparse` will warn on System Header and missed the real warning (because it'll stop after some threshold)

Reply:
Ideally, I think we'll make a mirror for sparse in git org, run action for that one and pull the artifacts into this project.

I can make it run under my namespace, but tying it to my repo doesn't look right.

Reply:
Don't we have such a pipeline already? https://dev.azure.com/git/git/_build?definitionId=10

Reply:
The static analysis job is running on `ubuntu-latest` which is `18.04`.

We need to build sparse for Ubuntu `18.04` instead

Reply:
I can modify it to run in Ubuntu 16.04, though

Reply:
OK, I looked into travis job, it look like we're running on Ubuntu 16.04 overthere.
I'll update GitHub Action to ubuntu-16.04.

Sorry for bugging you.

Reply:
No worries, and sorry for being so unresponsive!